src/hw9/Main.java

```diff
@@ -3,17 +3,17 @@
 import static hw9.ArrayValueCalculator.doCalc;
 
 public class Main{
-    public static void main(String[] args) {
+    public static void main (String[] args){
```

Comment:
Formatting `public static void main (String[] args){` should be as follows: `public static void main(String[] args) {`